### ISSUE TITLE
Analytics module - price/time quote

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/atla_digital_library#79]
![Image](https://user-images.githubusercontent.com/29311858/202256608-01791d5b-c5cf-41d7-b83c-a9463e0ac397.png)

This also uses Google Analytics to capture views and downloads on individual works and collections.

Work may come from partial Hyrax work.

**Comment by crisr15** [comment]
Christy, can we close this?  New analytics functionality was completed in hyrax and you will need a hyrax upgrade to bring in this work.

**Comment by ckarpinski** [comment]
Yes,I think all this ever was was asking what it would cost to develop this, and we decided not to do it.
